spec/src/main/asciidoc/chapters/introduction/introduction.asciidoc

```diff
@@ -14,14 +14,101 @@
 
 == Introduction
 
+:Jakarta-Persistence: https://jakarta.ee/specifications/persistence/
+:Jakarta-Data: https://jakarta.ee/specifications/data/
+:Jakarta-NoSQL: https://jakarta.ee/specifications/nosql/
+
+Jakarta Query defines an object-oriented query language designed for use with
+link:{Jakarta-Persistence}[Jakarta Persistence], link:{Jakarta-Data}[Jakarta Data],
+and link:{Jakarta-NoSQL}[Jakarta NoSQL], with:
+
+- a core language which can be implemented by Jakarta Data and Jakarta NoSQL
+providers using non-relational datastores, and
```

Comment:
It's probably best not to limit the Jakarta Data case for the core language to non-relational because a Jakarta Data provider for relational databases might use this query language for an implementation based on JDBC or SQL.  I don't know if any providers will actually do that, but it's possible and should be allowed for if anyone wanted to.  I suppose another possibility would be a Data provider based on Jakarta Persistence, but choosing to limit the user to the core query language.
```suggestion
- a core language which can be implemented by Jakarta Data and Jakarta NoSQL
providers, and
```